src-plugins/vtkDataMesh/readers/vtkDataMeshReader.cpp

```diff
@@ -89,28 +90,124 @@ bool vtkDataMeshReader::read(const QString& path) {
             return false;
         }
 
-        try
-        {
-            dataSet->Read(path.toLocal8Bit().constData());
```

Comment:
Well, i changed it to the vtkPolyDataReader and vtkXMLPolyDataReader because we need it to extracts header data. You mean  `vtkMetaSurfaceMesh::ReadVtkFile` and `vtkMetaSurfaceMesh::ReadVtpFile` i imagine. I can try indeed! :)